### PR TITLE
[16.0][MIG] base_transaction_id: Migration to 16.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Note: this migration is based on this [PR](https://github.com/OCA/account-reconcile/pull/450)

### CODE REVIEW

File: base_transaction_id/i18n/en.po [hunk 1]
```diff
@@ -1,55 +0,0 @@
-# Translation of Odoo Server.
-# This file contains the translation of the following modules:
-# * base_transaction_id
-#
-# Translators:
-msgid ""
-msgstr ""
-"Project-Id-Version: bank-statement-reconcile (9.0)\n"
```
Comment (leemannd) [inline base_transaction_id/i18n/en.po]: Any reason to delete a translation file during the migration?

**Review by leemannd — APPROVED** [review]
Thank you

**Review by yankinmax — APPROVED** [review]

**Review by santostelmo — APPROVED** [review]

**Review by StephaneMangin — APPROVED** [review]

**Comment by RodrigoBM** [comment]
issue: https://github.com/OCA/account-reconcile/issues/497

**Comment by RodrigoBM** [comment]
@dzungtran89 can you change name PR, Thanks

- For being easily searchable and avoiding to duplicate efforts, please name the pull request following this pattern: [16.0][MIG] <module>: Migration to 16.0

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

**Review by RodrigoBM — APPROVED** [review]

**Comment by RodrigoBM** [comment]
@leemannd can you merge it?, thanks

**Comment by dzungtran89** [comment]
Hello @RodrigoBM , thanks for the review, I updated the PR's title

**Comment by leemannd** [comment]
Hello @RodrigoBM I'm not PSC on this repository so I don't have merge rights.  Has someone from @OCA/accounting-maintainers time to have a quick look about it?

**Comment by RodrigoBM** [comment]
@leemannd Im sorry, I thought you had permissions

**Pull request merged**